src/components/SimpleTable.js

```diff
@@ -35,21 +35,38 @@ const OnClickTextCell = ({ text, style, onClick, keyProp }) => {
 };
 
 //Column component that renders the column row based on column headers
-const ColumnRow = ({columnHeaders, baseStyle, columnStyle}) => {
+const ColumnRow = ({columnHeaders, baseStyle, columnStyle, sort, onSort}) => {
   const rowStyle = Object.assign({}, baseStyle, columnStyle);
-  return div({style: rowStyle, key: `column-row-container`, role:'row'}, columnHeaders.map((header) => {
+  return div({style: rowStyle, key: `column-row-container`, role:'row'}, columnHeaders.map((header, colIndex) => {
     const {cellStyle, label} = header;
     //style here pertains to styling for individual cells
     //should be used to set dimensions of specific columns
-    return div({style: cellStyle, key: `column-row-${label}`}, [label]);
+    return div({style: cellStyle, key: `column-row-${label}`, className: 'column-header'}, [
+      header.sortable && onSort
+        ? div({
+          style: Styles.TABLE.HEADER_SORT,
+          key: "data_id_cell",
+          className: 'cell-sort',
+          onClick: () => {
```

Comment:
For the `NewResearcherConsole` having "Actions" as a sortable column throws errors when you click on the header. Since 'status' already provides that contextual information, we don't need "Actions" to be sortable at all.

![Kapture 2022-02-08 at 07 10 03](https://user-images.githubusercontent.com/6664778/152985063-8adea3c1-0a9a-47f2-9ee5-0f69f47fad8b.gif)

Reply:
I think I pushed up a fix for this now! Thanks for the catch.

I'm going to also try to find the styles for that revise button, heh!